Makefile

```diff
@@ -45,13 +45,13 @@ test-coverage-all:
 
 integration-unix: build
 	$(info INFO: Starting build $@)
-	commander test commander_unix.yaml
+	./commander test commander_unix.yaml
 
 integration-linux: build
 	$(info INFO: Starting build $@)
 	./integration/setup_unix.sh
-	commander test commander_unix.yaml
-	commander test commander_linux.yaml
+	./commander test commander_unix.yaml
```

Comment:
This was by intention. With this line you would use the dev-build for integration testing. Normally in the path should be a stable version of commander, especially in CI environments.

Reply:
Oh I see, very meta! I'm not using GOPATH so I'll just add a stable build to my personal bin/.